web-wallet/src/lib/components/Stake/Stake.svelte

```diff
@@ -113,18 +122,74 @@
 	$: fee = formatter(luxToDusk(luxFee));
 	$: maxSpendable = deductLuxFeeFrom(spendable, luxFee);
 	$: minStake = maxSpendable > 0 ? Math.min(defaultMinStake, maxSpendable) : defaultMinStake;
+
+	let hideStakingNoticeNextTime = false;
+
+	function getWizardSteps () {
+		if (flow === "stake") {
+			return hideStakingNotice ? 3 : 4;
+		}
+
+		return 2;
+	}
 </script>
 
 <div class="operation">
-	<Wizard steps={flow === "stake" ? 3 : 2} let:key>
+	<Wizard steps={getWizardSteps()} let:key>
 		{#if flow === "stake"}
+			{#if !hideStakingNotice}
+				<!-- STAKING NOTICE STEP -->
```

Comment:
I know someone had removed these comments before – I have added them back, as the steps change based on different conditions and it's hard to reason about the code, if there's not an easy way to know what step the developer is looking at.